### PR TITLE
Remove match_rule and f attributes.

### PR DESCRIPTION
**Pull request by jsha**
They're no longer used.

cc @semenko for review.

### CODE REVIEW

**Comment by semenko** [comment]
Good call! Cursory LGTM -- I'll test this more over the ~weekend.

@blockoperation had a lot of related changes (#1775) for inspiration at https://github.com/blockoperation/https-everywhere/commits/chromium-memory-usage-v2

File: src/chrome/content/code/ApplicableList.js [hunk 1]
```diff
@@ -233,37 +218,20 @@ ApplicableList.prototype = {
...
     if (rule.active && type == 'inactive')
-      type = 'moot';
-    
+      type = 'active';
+
     // set the icon
     var image_src;
     if (type == 'active') image_src = 'tick.png';
     else if (type == 'inactive') image_src = 'cross.png';
-    else if (type == 'moot') image_src = 'tick-moot.png';
```
Comment (semenko) [inline src/chrome/content/code/ApplicableList.js]: please rm skin/tick-moot.png

**Comment by semenko** [comment]
Ok, just a few outstanding `moot` references to clean up:

```
chrome/content/code/AndroidUI.jsm
chrome/skin/https-everywhere.css
chrome/skin/tick-moot.png
```

**Comment by jsha** [comment]
Fixed the issues you mentioned. Thanks!

**Comment by semenko** [comment]
:laughing:  lingering moots

**Pull request merged**